packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -949,52 +1054,83 @@ function makeTitle(webxdcInfo: T.WebxdcMessageInfo, chatName: string): string {
   }${truncateText(webxdcInfo.name, 42)} – ${chatName}`
 }
 
-function partitionFromAccountId(accountId: number) {
-  return `persist:webxdc_${accountId}`
+function partitionFromId(id: number | string) {
+  return `persist:webxdc_${id}`
 }
 
-function sessionFromAccountId(accountId: number) {
-  return session.fromPartition(partitionFromAccountId(accountId), {
+function sessionFromId(id: number | string) {
+  return session.fromPartition(partitionFromId(id), {
     cache: false,
   })
 }
```

Comment:
I think it would be more clear to have separate methods for "get partition from account id" and "get partition for app id".

But I also don't quite understand why each internet enabled app needs it's own session, wouldn't it be enough to just have two sessions per account? one with internet and one without?

we could give the maps scheme to all internet enabled webxdcs, internet enabled webxdcs won't work anyway because of the DNS block in chromium...
if we want them to work, then we could think about overwriting the "https" scheme if that is possible instead of creating a new maps scheme.